packages/fxa-settings/src/components/AppLayout/index.tsx

```diff
@@ -26,6 +26,7 @@ export const AppLayout = ({ children }: AppLayoutProps) => {
           <div
             id="alert-bar-root"
             data-testid="alert-bar-root"
+            className="fixed z-10"
```

Comment:
If header ends up getting `z-10`, this should be bumped to `z-20` just to be safe